serverless.yml

```diff
@@ -35,6 +37,34 @@ functions:
           path: /api/v1/{id}
           method: DELETE
 
+  GetHandler:
```

Comment:
Thank you sir! I've replaced it with GetItem and others with their respective functionality. So that, whenever I want to use this CRUD in future, like for users or books or anything else, I don't have to create separate functions for them. I can use my existing functions by passing their respective arguments. Please review and correct me if I'm making mistake in this case. Thank you!